### PR TITLE
Add single node wazuh-manager setup

### PR DESCRIPTION
**Pull request by Wasaac**

### CODE REVIEW

**Comment by oneswig** [comment]
Thanks @Wasaac I'll try it out this week

File: vars/wazuh-manager.yml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+# Elasticsearch node.name
+# Also used in certificate file paths
+elasticsearch_node_name: "{{ inventory_hostname }}"
+
+# Elasticsearch network.host
+elasticsearch_network_host: "{{ wazuh_manager_ip }}"
+
+# Modify major versions at your own peril
+elastic_stack_version: "7.10.0"
```
Comment (jovial) [inline vars/wazuh-manager.yml]: I got an error with a version mistach:

"Plugin wazuh [7.10.0] is incompatible with kibana version [7.10.2]". Possibly depends on version of role? Should we pin the version in requirements.yml? I noticed you did that for sms lab but requirements.yml is missing from this PR. Just wondered if this was a mistake or intentional.

Comment (Wasaac) [reply vars/wazuh-manager.yml]: Mistake on my part! I'd updated smslab but forgot to port the fix. You're right that should be `elastic_stack_version: "7.10.2"`
The `requirements.yml` entry with the role was already included [here](https://github.com/stackhpc/kayobe-ops/pull/8).

**Comment by oneswig** [comment]
This approach appears to be working at various sites, I'll merge it and we can iterate on it from there.  Thanks @Wasaac

**Pull request merged**